lib/provider/boku.py

```diff
@@ -33,6 +33,10 @@ def create(self, request):
         cleaned = form.cleaned_data
         transaction = cleaned['param']
 
+        # Check the signature shows it was from Boku.
+        log.info('Checking signature')
+        check_sig(request.DATA)
```

Comment:
if this raises a VerificationError it's probably better to log it and return a 400, no? We'd probably still want the error to go to Sentry though.